src/repositories/linked-accounts/linked-account.repository.ts

```diff
@@ -23,12 +23,11 @@ export class LinkedAccountRepository {
   async findOneByUserIdAndAccountName(query: {
     account: LinkedAccount['account'];
     user_id: LinkedAccount['user_id'];
-    network: LinkedAccount['network'];
   }) {
     return await this.linkedAccountModel.findOne({
       ...query,
       status: 'verified',
-    } satisfies LinkedAccount);
+    } satisfies Partial<LinkedAccount>);
```

Comment:
Doesn't this already have type checking so `satisfies` wouldn't be required?

Reply:
No, mongoose has terrible type-checking, this could fail at runtime due to the spread operator (possibly?) - I can't remember exactly but it was needed for some reason.

Reply:
I'm not familiar enough with mongoose to know if it is broken or not. I think it is worth double checking. Can you test this?

Reply:
![image](https://github.com/spknetwork/acela-core/assets/7293419/d59f4e9e-35d7-4d28-b872-433acbb3d85c)

Reply:
Weird. I see. As long as there is nothing missed with added the `LinkedAccount` type to `linkedAccountModel`, then I think this is fine.